.github/dependabot.yml

```
@@ -9,3 +10,24 @@ updates:
     labels:
       - area/dependencies
       - area/ci
+  # keycloak UI dependencies
+  - package-ecosystem: npm
+    directory: /themes/src/main/resources/theme/keycloak/common/resources/
```

There is a trailing slash here that can be removed.
```suggestion
    directory: /themes/src/main/resources/theme/keycloak/common/resources
```